examples/mnist/run.py

```diff
@@ -177,15 +177,24 @@ def receive_output(self, logits: tf.Tensor) -> tf.Operation:
   # avoid re-training each time
   cache_updater, params = tfe.cache(params)
 
+  w0, b0, w1, b1 = params
+  initializer_w0 = tf.keras.initializers.Constant(w0)
+  initializer_b0 = tf.keras.initializers.Constant(b0)
+  initializer_w1 = tf.keras.initializers.Constant(w1)
+  initializer_b1 = tf.keras.initializers.Constant(b1)
+  model = tfe.keras.Sequential([
+    tfe.keras.layers.Dense(512, batch_input_shape=(PredictionClient.BATCH_SIZE, 784), kernel_initializer=initializer_w0, 
+    bias_initializer=initializer_b0),
+    tfe.keras.layers.BatchNormalization(axis=1),
```

Comment:
how does this perform normalization if the batch statistics weights haven't been initialized?  we can't add layers here without changing the way training happens in `ModelOwner._build_training_graph`

Reply:
Agree with this, had removed it from my other implementations but forgot to here. Thanks, will do so!